docs/getting-started/using-gnoland.md

```diff
@@ -0,0 +1,543 @@
+---
+id: using-gnoland
+---
+
+## Using `gnoland`
+
+## Overview
+
+In this tutorial, you will learn how to use the `gnoland` binary. This guide covers
+various available subcommands and flags to help you configure, manage, and spin up
+a local gno.land node.
+
+## Prerequisites
+
+- **Git**
+- **`make` (for running Makefiles)**
+- **Go 1.21+**
+- **Go Environment Setup**: Ensure you have Go set up as outlined in the [Go official installation documentation](https://go.dev/doc/install) for your environment
+
+## Installation
+
+To install the `gnoland` binary, clone the Gno monorepo:
+
+```bash
+git clone https://github.com/gnolang/gno.git
+```
+
+After cloning the repo, go into the `gno.land/` folder, and use the existing
+Makefile to install the `gnoland` binary:
+
+```bash
+cd gno.land
+make install.gnoland
+```
+
+To verify that you've installed the binary properly and that you are able to use
+it, run the `gnoland` command:
+
+```bash
+gnoland --help
+```
+
+If everything was successful, you should get the following output:
+
+```bash
+USAGE
+  <subcommand> [flags] [<arg>...]
+
+starts the gnoland blockchain node.
+
+SUBCOMMANDS
+  start    run the full node
+  secrets  gno secrets manipulation suite
+  config   gno config manipulation suite
+  genesis  gno genesis manipulation suite
+```
+
+If you do not wish to install the binary globally, you can build it with the
+following command from the `gno.land/` folder:
+
+```bash
+make build.gnoland
+```
+
+And finally, run it with `./build gnoland`.
+
+Let's dive deeper into each of the subcommands and see how we can use them.
+
+## Running a Local Node
+
+Using the `gnoland start` command allows you to run a local gno.land node.
+The node can be started in two main ways:
+- With `lazy` mode
+- With manual configuration
+
+Let's see how we can start the node lazily for the quickest setup.
+
+### Lazy Node Initialization
+
+The simplest way to spin up a local node is to use the following command:
+
+```bash
+gnoland start --lazy
+```
+
+![gnoland-start-lazy](../assets/getting-started/using-gnoland/gnoland-start-lazy.gif)
+
+This command will generate all necessary files for you, including the `genesis.json`
+file and node secrets, which include the validator & node private keys.
+By default, the node will start listening on `localhost:26657`.
+
+#### Flags
+- `-chainid dev` - the ID of the chain.
```

Comment:
Maybe just link to a reference page so they same information mustn't be maintained in two places?